api/v2/api.proto

```diff
@@ -160,13 +160,13 @@ message VerifyPasswordResp {
   bool not_found = 2;
 }
 
-message GetVehiclePrivilegeTokenReq {
-  string user_id = 1;
+message GetPrivilegeTokenReq {
+  string user_eth_address = 1;
   string  vehicle_token_id = 2;
```

Comment:
We can worry about this later but note that this field is still `vehicle_token_id`.